server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java

```diff
@@ -378,7 +378,8 @@ public void testCopyToDirectoryOnPartitionedTableWithPartitionClause() throws Ex
 
     @Test
     public void testCopyToDirectoryOnPartitionedTableWithoutPartitionClause() throws Exception {
-        new Setup(sqlExecutor).partitionTableSetup();
+        // Use more than one shard per partition to expose possible issues with the FileWriterCountCollector
+        new Setup(sqlExecutor).partitionTableSetup(2);
```

Comment:
Before this change we used a statement without CLUSTERED INTO clause.

Wasn't it already more than one shard? As far as I know, if value is not provided default is 
`num_shards = max(4, num_data_nodes * 2)`.

Ie master branch is equivalent of this change with `new Setup(sqlExecutor).partitionTableSetup(4)` or I'm missing something?

Reply:
Nevermind, it affects docs distribution, so with 4 we most likely had `_id1 % 4 = _id2 % 4` -> single shard per partition and now we have `_id1%2 != _id2%2` -> 2 shards par partition.

Maybe can elaborate on
> Use more than one shard per partition 

And make it

> Use 2 shards to ensure that each partition's documents go into different shards and hence each partition will have more than one shard.

Reply:
Thx, adjusted the comment.